client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/hooks/use-recipe.ts

```diff
@@ -67,14 +67,8 @@ const useRecipe = (
 		[]
 	);
 
-	const preselectedDesign = useMemo(
-		() =>
-			allDesigns?.designs.find( ( design ) =>
-				design.is_virtual
-					? design.recipe?.slug === preselectedThemeSlug
-					: design.slug === preselectedThemeSlug
-			),
-		[ allDesigns ]
+	const preselectedDesign = allDesigns?.designs?.find(
```

Comment:
Yup, let's try this out. After looking at the endpoint code, it's definitely odd that `designs` can be anything different from an array. 🤔